cli/command/container/stats.go

```diff
@@ -125,13 +127,16 @@ func runStats(ctx context.Context, dockerCli command.Cli, opts *statsOptions) er
 	}
 
 	if showAll {
+		span := trace.SpanFromContext(ctx)
+
 		// If no names were specified, start a long running goroutine which
 		// monitors container events. We make sure we're subscribed before
 		// retrieving the list of running containers to avoid a race where we
 		// would "miss" a creation.
 		started := make(chan struct{})
 		eh := command.InitEventHandler()
 		eh.Handle(events.ActionCreate, func(e events.Message) {
+			span.AddEvent(string(events.ActionCreate), trace.WithAttributes(attribute.String("ID", e.ID)))
```

Comment:
Note to self; forgot to update this one to use `e.Actor.ID` (same as `e.ID`, but that's officially deprecated)

Reply:
done 👍